### PR TITLE
LPS-165702 Adapt ModalAddField to support the creation of a Multi-Select Picklist field

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://issues.liferay.com/browse/LPS-165702

### CODE REVIEW

**Comment by danvb112** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx [hunk 1]
```diff
@@ -325,30 +330,34 @@ export default function ObjectFieldFormBase({
...
-					}}
-					options={pickLists}
-					required
-					value={
-						validListTypeDefinitionId ? selectedPicklist?.name : ''
-					}
-				/>
-			)}
+			{values.businessType === 'Picklist' ||
+				(values.businessType === 'MultiselectPicklist' && (
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx:334]: Hi @danvb112. I think this way can make it more clear:

```suggestion
			{(values.businessType === 'Picklist' ||
				values.businessType === 'MultiselectPicklist') && (
```

**Review by alinedoleron — CHANGES_REQUESTED** [review]
Just a small adjustment =)

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx (hunk 1 above)
Comment (danvb112) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx]: Thanks! @alinedoleron

**Comment by danvb112** [comment]
ci:test:sf

**Review by alinedoleron — APPROVED** [review]

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**